pages/home.tsx

```diff
@@ -0,0 +1,274 @@
+import { NextPage } from 'next'
+import Image from 'next/image'
+
+import { Box, Button, Container, Typography, useTheme } from '@mui/material'
+import useMediaQuery from '@mui/material/useMediaQuery'
+
+import HandshakeOutlinedIcon from '@mui/icons-material/HandshakeOutlined'
+import Groups2OutlinedIcon from '@mui/icons-material/Groups2Outlined'
+import VolunteerActivismOutlinedIcon from '@mui/icons-material/VolunteerActivismOutlined'
+
+import CardRowContainer from 'components/cards/CardRowContainer'
+import CardOne from 'components/cards/CardOne'
+
+import HeroLines from '../public/images/homeHeroLines.svg'
+import HeroImage from '../public/images/seattle_4.jpg'
+
+interface Props {}
+
+const Home: NextPage<Props> = () => {
+  const theme = useTheme()
+  const palette = theme.palette
+
+  const loader = ({ src, width, quality }) => {
+    return `${process.env.NEXT_PUBLIC_BASE_PATH}${src}\?w=${width}&q=${
+      quality || 75
+    }`
+  }
+
+  const isMediumScreen = useMediaQuery('(max-Width:900px)')
```

Comment:
Hey I noticed a lot of css is made to make the website responsive. Could you look at [stack]( https://mui.com/material-ui/react-stack/) and [grid](https://mui.com/material-ui/react-grid2/)
could automatically adjust for screen size and solve the issue. 


Especially grid, here's a [video](https://youtu.be/jrgiTFwvglw?t=164) on it.

Reply:
oh like what @mistyb01  did [here](https://github.com/openseattle/open-seattle-website/pull/18/files#diff-7321353cedfba627ec35e731b0b944c9cdca5110bd271896aab3ba468a51b1a5R75)

Reply:
Thanks Santina - yeah, it does feel like a lot of CSS. I was struggling to get all the graphics and the shadows to work with the stack component. Is there a specific place you would recommend implementing it?

Reply:
mm stack should only be a wrapper around the components and not impacting the components themselves. Could you share what you had and we can debug together?